plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk

```diff
@@ -33,8 +33,7 @@ algorithm force(ForceLayoutProvider) {
     features multi_edges, edge_labels
     preview images/force_layout.png
     supports org.eclipse.elk.priority = 1
-	   documentation 
-	       "Priorities set on nodes determine the order in which connected components are placed:
+    documentation "Priorities set on nodes determine the order in which connected components are placed:
```

Comment:
Why is the newline removed here, but added in `Disco.melk`?

Reply:
Are you maybe confusing `documentation` and `description`? 

`documentation` should (usually) refer to some markdown file, e.g. `@docs/myOption.md`. As such, I thought one single line should be fine. 
However, there are some rare cases where text is provided directly.

Reply:
Oh... so I am. Never mind me, then. 🤦